README.md

```diff
@@ -19,6 +19,21 @@ You will find some [basic instructions](https://guide.openrndr.org/setUpYourFirs
 
 To create runnable jars for a platform different from the platform you use to build one uses `./gradlew jar --PtargetPlatform=<platform>`. The supported platforms are `windows`, `macos`, `linux-x64` and `linux-arm64`. 
 
+## Updating OPENRNDR, ORX and other dependencies
+
+The openrndr-template depends on various packages including the core [openrndr](https://github.com/openrndr/openrndr/) and the [orx](https://github.com/openrndr/orx/) extensions.
+
+The version numbers of these dependencies are specified in the [libs.versions.toml](https://github.com/openrndr/openrndr-template/blob/master/gradle/libs.versions.toml) file. If you want to learn about file format visit the [Gradle documentation](https://docs.gradle.org/current/userguide/platforms.html#sub:conventional-dependencies-toml) website.
```

Comment:
Maybe better to point to the local libs.versions.toml file? Then the README.md can be used to navigate the codebase.